CHANGELOG

```diff
@@ -4,7 +4,7 @@ v3.0.0 (29 Jun 2020)
 ------------------------
 Contributor Pages
 * Fix #9417: No difficulty level to select for questions while creating question suggestion (#9436)
-* Fix #9375: Adds custom entity context for the translations in community dashboard (#9386)
+* Fix #9375: Adds custom entity context for the translations in contributor dashboard (#9386)
```

Comment:
We should not update the CHANGELOG, I think. Can you please revert these changes?